src/components/TextListItem.tsx

```diff
@@ -135,15 +134,14 @@ export const TextListItem: NamedExoticComponent<Props> & {
         </ListItem.Content>
       )}
 
-      {rightIcon ||
-        (rightImage && !!picture?.url ? (
-          <Image
-            source={{ uri: picture.url }}
-            style={[styles.smallImage, withCard && styles.withBigCardStyle]}
-            borderRadius={withCard && normalize(8)}
-            containerStyle={styles.smallImageContainer}
-          />
-        ) : undefined)}
+      {(rightIcon || rightImage) && !!picture?.url ? (
```

Comment:
the condition changed accidentally? if a rightIcon is given, it should be rendered. everything else was the "else" part, which is now changed?

Reply:
I'm sorry. I misunderstood the logic here. I'll undo the changes.